apps/playground-web/tsconfig.json

```diff
@@ -1,13 +1,16 @@
 {
   "extends": "@dicedb/typescript-config/nextjs.json",
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
Can you do this in `typescript-config/nextjs.json`?

Reply:
Hello @AjayPoshak did you mean to remove these in

-     apps/playground-web/tsconfig.json
-     packages/ui/tsconfig.json

 and add here
 
- typescript-config/nextjs.json

Reply:
Yes, we should have common config placed there

Reply:
Thank you for the feedback. please check now